src/main/java/org/springframework/samples/petclinic/model/Pet.java

```diff
@@ -119,4 +118,11 @@ public void addVisit(Visit visit) {
         visit.setPet(this);
     }
 
+    public boolean equals(Object other){
```

Comment:
when using boolean it is suggested to use is, has, should using the P/A/HC/LC pattern

Reply:
This is a standard Java method for all Java objects, in order to override it you must follow the same signature. Check [here](https://www.geeksforgeeks.org/overriding-equals-method-in-java/) for more details.